sql/src/main/java/io/crate/action/sql/TransportBaseSQLAction.java

```diff
@@ -403,7 +403,7 @@ public SQLActionException buildSQLActionException(Throwable e) {
                 message = "Error in " + e.getClass().getSimpleName();
             }
         } else {
-            message = e.toString();
+            message = e.getClass().getSimpleName() + ": " + message;
```

Comment:
no, seems like they already do some form of contains (that's the default for expectMessage)